portlets/src/main/webapp/vue-app/engagement-center/components/common/ResultNotFound.vue

```diff
@@ -49,26 +49,69 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         fas fa-trophy
       </v-icon>
     </div>
-    <div class="d-flex flex-column justify-center align-center py-12">
-      <span class="text-header-title font-weight-regular">
-        {{ $t('programs.details.programDeleted') }}
-      </span>
+    <div class="d-flex flex-column justify-center align-center py-12 col-sm-8 mx-auto">
+      <p class="text-header-title font-weight-regular mb-8">
+        {{ messageTitle }}
+      </p>
+      <p class="text-header-title font-weight-regular">
+        {{ messageInfoOne }}
+      </p>
+      <p 
+        v-if="sanitizedHtml" 
+        class="text-header-title font-weight-regular" 
+        v-sanitized-html="sanitizedHtml"> 
+      </p>
       <v-btn
-        class="btn btn-primary mt-8"
-        :href="programListLink">
+        v-if="buttonText" 
+        class="btn btn-primary my-8"
+        @click="$emit('button-event')"
+        :href="url">
```

Comment:
please follow attributes order defined here https://eslint.vuejs.org/rules/attributes-order.html

```suggestion
        v-if="buttonText"
        :href="url"
        class="btn btn-primary my-8"
        @click="$emit('button-event')">
```